### ISSUE TITLE
failures downloading (8.5 38556) from backblaze B2 on Windows 11, unhandled exception during invoke (overflow)

### ISSUE THREAD

**Issue by twdeckard** [iterate-ch/cyberduck#13904]
Tried numerous attempts to download a 30GB zip file on a reasonable bandwidth connection 5-30MBps.

Download appeared to proceed normallly.   Download would "accelerate".  stored files never got close to 30GB size. 

Errors were 
```
2022-11-26 12:07:18,888 [timer-1] ERROR AsyncController - Unhandled exception during invoke
cli.System.OverflowException: Value was either too large or too small for an Int32.
2022-11-26 13:43:03,476 [timer-1] ERROR AsyncController - Unhandled exception during invoke
cli.System.OverflowException: Value was either too large or too small for an Int32.
```

Never successful.

**Comment by Instinct64** [comment]
I'm having a similar issue with large files, when my router is configured to route some traffic via VPN with VPN Policy. When VPN is completely disabled it doesn't seem to occur. Nevertheless, Cyberduck ends up in a bit of a state for some reason.

The transfer get's out of sync with reality, and often starts saturating my connection with traffic but never finishing the download. Screenshot is what happens after I press "Stop" and wait for it to wind up.

![image](https://user-images.githubusercontent.com/7610500/224115960-e48750a8-7951-4cb7-aff2-1e49de72ab3c.png)

Cyberduck in it's "stuck" state. The transfer window is "static" and the data isn't changing.
![image](https://user-images.githubusercontent.com/7610500/224116645-1fdc048a-04eb-4ab6-b9c8-10162b63c1fa.png)

But in the background. Cyberduck is busy downloading data (at least for a while longer)
![image](https://user-images.githubusercontent.com/7610500/224116783-6f057931-c274-4c4a-8e1f-7febc22d0463.png)

For clarity. The transfer never finishes. Using stop and resume can sometimes allow the transfer to progress a little further.

**Comment by Instinct64** [comment]
@AliveDevil - Any chance you're able to accept this as an issue?

**Comment by AliveDevil** [comment]
Will check, thanks for reminding.

**Comment by AliveDevil** [comment]
So, this is only reproducible with failing transfers - a passing transfer (without network or service interrupts) doesn't observe this behavior (unfortunately).

I identified an issue, that may or may not resolve the issue here, though whether this makes any difference needs to be seen.

**Comment by AliveDevil** [comment]
This particular issue will be closed with #14439 - that is the progress bar and progress state "seem" to be stuck, due to an issue that came up internally.

This is a workaround, and the underlying issue (that Cyberduck keeps incrementing the "transferred"-amount) will be handled with a later PR.

**Comment by twdeckard** [comment]
unsubscribe
